addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -2143,17 +2142,6 @@ def test_routing_loop_follower_alias(self):
         self.assertNotSentEmail()
         self.assertFalse(bool(self._new_msgs))
 
-        # simulate stupid email providers that rewrites msg_id -> thanks to
-        # a custom header, it is rejected as already managed by mailgateway
-        self._mails = last_mail
-        with RecordCapturer(self.env['mail.test.ticket'], []) as capture_ticket, \
-             RecordCapturer(self.env['mail.test.gateway'], []) as capture_gateway:
-            self._reinject(force_msg_id='123donotnamemailjet456')
```

Comment:
Do we have test covering "wrong in-reply-to but original message_id in references" ?

Reply:
It doesn't look like we do